protected/views/gp/_form.php

```diff
@@ -20,42 +23,117 @@
-        <div class="large-3 column"><?php echo $form->labelEx($model, 'last_name'); ?></div>
-        <div class="large-4 column end">
+        <div class="column"><?php echo $form->labelEx($model, 'last_name'); ?></div>
+        <div class="column end">
             <?php echo $form->textField($model, 'last_name', array('size' => 30, 'maxlength' => 100)); ?>
             <?php echo $form->error($model, 'last_name'); ?>
         </div>
       </div>
 
       <div class="row field-row">
-        <div class="large-3 column"><?php echo $form->labelEx($model, 'primary_phone'); ?></div>
-        <div class="large-4 column end">
+        <div class="column"><?php echo $form->labelEx($model, 'primary_phone'); ?></div>
+        <div class="column end">
             <?php echo $form->telField($model, 'primary_phone', array('size' => 15, 'maxlength' => 20)); ?>
             <?php echo $form->error($model, 'primary_phone'); ?>
         </div>
       </div>
+
+      <div class="row field-row">
+        <div class="column">
+            <?php echo $form->labelEx($model, 'Role'); ?>
+        </div>
+        <div class="column end">
+            <?php echo $form->error($model, 'contact_label_id'); ?>
+            <?php
+            $this->widget('zii.widgets.jui.CJuiAutoComplete', array(
+                'name' => 'contact_label_id',
+                'id' => 'autocomplete_contact_label_id',
+                'source' => "js:function(request, response) {
+                                $.getJSON('/gp/contactLabelList', {
+                                   term : request.term
+                                }, response);
+                        }",
+                'options' => array(
+                    'select' => "js:function(event, ui) {
+                                removeSelectedContactLabel();
+                                addItem('selected_contact_label_wrapper', ui);   
+                                $('#autocomplete_contact_label_id').val('');
+                                return false;
+                                }",
+                    'response' => 'js:function(event, ui){
+                          if(ui.content.length === 0){
+                            $("#no_contact_label_result").show();
+                          } else {
+                            $("#no_contact_label_result").hide();
+                          }
+                        }',
+                ),
+                'htmlOptions' => array(
+                    'placeholder' => 'search Roles',
+                ),
+            ));
+            ?>
+        </div>
+      </div>
+      <div id="selected_contact_label_wrapper" class="row field-row <?php echo $model->label ? '' : 'hide' ?>">
+        <div class="column selected_contact_label end alert-box">
+          <span class="name">
+            <?php echo isset($model->label) ? $model->label->name : ''; ?>
+          </span>
+          <a href="javascript:void(0)" class="remove right">remove</a>
```

Comment:
Is anyone else finding that the "remove" link isn't blue until it is highlighted?

Reply:
I see it too. Probably some cascading CSS formatting clashing with the link colour.

Reply:
Yep ".ui-widget-content a" is setting the colour over the top of "a"

Reply:
yep